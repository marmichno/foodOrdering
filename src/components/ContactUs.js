import {motion} from 'framer-motion';
import {
    withScriptjs,
    withGoogleMap,
    GoogleMap,
    Marker,
  } from "react-google-maps";
import {AiFillPhone} from 'react-icons/ai'
import {AiOutlineMail} from 'react-icons/ai'

const containerVariants = {
  hidden: {
      x: 2000
  },
  visible: {
      x: 0,
      transition: {
        type: 'spring',
        delay: 0.5,
        mass: 1,
        damping: 12
      }
  },
  exit: {
      x:'-100vw',
      transition: {ease: 'easeInOut', duration:0.5}
  }
}

export const ContactUs = () => {

    const API_KEY = 'AIzaSyDmpWYJAee_72ryefb9BT-aqcwF2Cbc_mY';

    const MapWithAMarker = withScriptjs(withGoogleMap(props =>
        <GoogleMap
          defaultZoom={15}
          defaultCenter={{ lat: 50.220860, lng: 18.669770 }}
        >
          <Marker
            position={{ lat: 50.220860, lng: 18.669770 }}
          />
        </GoogleMap>
      ));

    return(
        <div className="orderMainContainer">
            <motion.div
                variants={containerVariants}
                initial="hidden"
                animate="visible"
                exit="exit"
                className="contactUsContainer">
            <MapWithAMarker
            googleMapURL='https://maps.googleapis.com/maps/api/js?key=AIzaSyDmpWYJAee_72ryefb9BT-aqcwF2Cbc_mY&v=3.exp&libraries=geometry,drawing,places'
            loadingElement={<div style={{ height: `100%` }} />}
            containerElement={<div style={{ height: `80%`, width: `100%` }} />}
            mapElement={<div style={{ height: `100%`, width: '100%' }} />}
            />
                <div className="contactUsTextContainer">
                    <p><AiFillPhone/> 600 500 400</p>
                    <p><AiOutlineMail/> somemail@gmail.com</p>
                </div>
            </motion.div>
        </div>
    )
}