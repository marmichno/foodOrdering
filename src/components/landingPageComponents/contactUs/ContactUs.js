import {motion} from 'framer-motion';
import {AiFillPhone} from 'react-icons/ai'
import {AiOutlineMail} from 'react-icons/ai'

const containerVariants = {
  hidden: {
      x: 2000
  },
  visible: {
      x: 0,
      transition: {
        type: 'spring',
        delay: 0.5,
        mass: 1,
        damping: 12
      }
  },
  exit: {
      x:'-100vw',
      transition: {ease: 'easeInOut', duration:0.5}
  }
}

export const ContactUs = () => {

    return(
        <div className="contactUsMainContainer">
            <motion.div
                variants={containerVariants}
                initial="hidden"
                animate="visible"
                exit="exit"
                className="contactUsContainer">

                <div className="mapContainer"><h1>There will be map</h1></div>
      
                <div className="contactUsTextContainer">
                    <p><AiFillPhone/> 600 500 400</p>
                    <p><AiOutlineMail/> somemail@gmail.com</p>
                </div>
            </motion.div>
        </div>
    )
}