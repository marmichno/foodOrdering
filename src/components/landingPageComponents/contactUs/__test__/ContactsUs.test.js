import {ContactUs} from '../ContactUs';
import TestRenderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

test("matches snapshot", () => {
    const tree = TestRenderer.create(<ContactUs/>).toJSON();
    expect(tree).toMatchSnapshot();
});