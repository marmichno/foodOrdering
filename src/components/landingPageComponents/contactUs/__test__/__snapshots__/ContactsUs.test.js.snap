// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="contactUsMainContainer"
>
  <div
    className="contactUsContainer"
    style={
      Object {
        "transform": "translateX(2000px) translateZ(0)",
      }
    }
  >
    <div
      className="mapContainer"
    >
      <h1>
        There will be map
      </h1>
    </div>
    <div
      className="contactUsTextContainer"
    >
      <p>
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M885.6 230.2L779.1 123.8a80.83 80.83 0 0 0-57.3-23.8c-21.7 0-42.1 8.5-57.4 23.8L549.8 238.4a80.83 80.83 0 0 0-23.8 57.3c0 21.7 8.5 42.1 23.8 57.4l83.8 83.8A393.82 393.82 0 0 1 553.1 553 395.34 395.34 0 0 1 437 633.8L353.2 550a80.83 80.83 0 0 0-57.3-23.8c-21.7 0-42.1 8.5-57.4 23.8L123.8 664.5a80.89 80.89 0 0 0-23.8 57.4c0 21.7 8.5 42.1 23.8 57.4l106.3 106.3c24.4 24.5 58.1 38.4 92.7 38.4 7.3 0 14.3-.6 21.2-1.8 134.8-22.2 268.5-93.9 376.4-201.7C828.2 612.8 899.8 479.2 922.3 344c6.8-41.3-6.9-83.8-36.7-113.8z"
          />
        </svg>
         600 500 400
      </p>
      <p>
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M928 160H96c-17.7 0-32 14.3-32 32v640c0 17.7 14.3 32 32 32h832c17.7 0 32-14.3 32-32V192c0-17.7-14.3-32-32-32zm-40 110.8V792H136V270.8l-27.6-21.5 39.3-50.5 42.8 33.3h643.1l42.8-33.3 39.3 50.5-27.7 21.5zM833.6 232L512 482 190.4 232l-42.8-33.3-39.3 50.5 27.6 21.5 341.6 265.6a55.99 55.99 0 0 0 68.7 0L888 270.8l27.6-21.5-39.3-50.5-42.7 33.2z"
          />
        </svg>
         somemail@gmail.com
      </p>
    </div>
  </div>
</div>
`;
